processing/src/main/java/io/druid/query/aggregation/first/FloatFirstAggregatorFactory.java

```diff
@@ -172,9 +185,10 @@ public Object deserialize(Object object)
   }
 
   @Override
-  public Object finalizeComputation(Object object)
+  @Nullable
+  public Object finalizeComputation(@Nullable Object object)
   {
-    return ((SerializablePair<Long, Float>) object).rhs;
+    return object == null ? object : ((SerializablePair<Long, Float>) object).rhs;
```

Comment:
Double* variant of this class uses `object == null ? null` that is better

Reply:
done.